### DISCUSSION TITLE
Configure HTTP response headers

### DISCUSSION THREAD

**Discussion by AdiWehrli** [micronaut-projects/micronaut-core#8151] [Q&A]
Hello together

Is there a way one can configure _HTTP response headers_ (e.g. `Cross-Origin-Opener-Policy`, `X-Frame-Options`, `Feature-Policy` etc.) in Micronaut configuration through `application.yml`? I did not find any documentation on how to implement such an intent.

Maybe there is a property prefix similar to CSP `micronaut.views.csp.*`?

Or should I create an appropriate pull-request or an issue as recommended by [tchiotludo in AKHQ discussion](https://github.com/tchiotludo/akhq/discussions/1232#discussioncomment-3860227)?

Thanks very much in advance.

Best regards,
Adi

**Comment by yawkat** [comment]
we have an existing filter for CORS configuration, but nothing to set general headers from config. it sounds like a reasonable feature to have, so feel free to create an issue. we're currently doing some reorganization in the http modules, so please wait one or two weeks if you want to make a pr.

**Comment by AdiWehrli** [comment]
Hello @yawkat

Thanks very much for your reply. I already made a corresponding change for AKHQ (https://github.com/tchiotludo/akhq/pull/1235/files). I could adapt it to micronaut as well, later. 

Best regards, 
Adi
